src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: doc
+title: Make your PWA feel like a native app on Windows
+excerpt: One of the main challenges for web apps on Windows has always been that it is hard for your web app to “feel” like an app to a user.
+description: Learn about new advanced capabilities available in Edge and Windows that can raise the quality of your user experience and make your PWA feel native on Windows!
+date: 2021-08-05
+updatedDate: 2021-08-05
+trending: true
+featured: false
+image: docs/windows/pwa-api-documentation/shortcuts.png
+isDocumentation: true
+backUrl: "/docs"
+author:
+  name: PWA Builder documentation
+tags:
+  - docs
+  - Documentation
+  - Windows
+  - Microsoft Store
+  - APIs
+---
+
+One of the main challenges for web apps on Windows has always been that it is hard for your web app to “feel” like an app to a user. Because web apps couldn’t tie into things such as the native share sheet, Bluetooth or File Explorer and had to use custom UI and application code for these common UX actions, Web Apps did not provide the experience that users expect from applications. For example, how many times have you not shared something after hitting the share button on a website because it asked you to log in before you could share, you couldn’t just easily grab a link, or it didn’t have the service that you were wanting to share too? Or breaking things down even more, how many times have you not used a web app just because it wasn’t on your task bar or start menu?
+
+These are all very common user experience challenges that Web Apps have always encountered, but with the new Chromium based Edge, Project Fugu and PWABuilder, developers now have the APIs and tools they need to truly make their Progressive Web Apps feel like a native app on Windows.
```

Comment:
First time mentioning Project Fugu, yeah? If so, link to https://fugu-tracker.web.app/